xclim/subset.py

```diff
@@ -848,7 +903,7 @@ def subset_gridpoint(
     # Subset with specific start_dates and end_dates
     >>> prSub = subset.subset_time(ds.pr,lon=-75,lat=45, start_date='1990-03-13',end_date='1990-08-17')
     """
-
+    dist = None
```

Comment:
Why are we needing to initialize dist?

Reply:
Code style. I'll see if I can rearrange some things so as to not have to do this.

Reply:
Not a big issue i was merely curious